packages/insomnia-importers/package.json

```diff
@@ -8,21 +8,36 @@
   "bugs": {
     "url": "https://github.com/kong/insomnia"
   },
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
   "bin": {
-    "insomnia-import": "./bin/cli"
+    "insomnia-import": "./dist/src/cli"
   },
   "scripts": {
+    "bootstrap": "npm run build",
+    "lint": "eslint . --ext .js,.ts,.tsx --ignore-path='../../.eslintignore'",
+    "clean": "tsc --build tsconfig.build.json --clean",
+    "postclean": "rimraf dist",
+    "build": "tsc --build tsconfig.build.json",
     "test": "jest --silent"
   },
   "dependencies": {
-    "apiconnect-wsdl": "^1.8.29",
+    "apiconnect-wsdl": "^1.8.31",
     "change-case": "^4.1.1",
-    "commander": "^2.20.0",
+    "commander": "^7.1.0",
```

Comment:
Is a jump of 5 major versions necessary here? What has changed in the 5 versions that could error out?

Reply:
please consult the scratchpad where the reason for this change was described https://docs.google.com/document/d/1nDYsQnWMHRNNIzhwMjHjfxGJf3lAO_zuumGwhi89jIo/edit#